src/views/HomeView.jsx

```diff
@@ -1,6 +1,6 @@
 import React from "react";
-import { Link } from "react-router-dom";
 import { UsersList } from "../state/use-users-list";
+import Users from "../components/HomeViewUsers";
```

Comment:
"View" is an indicator of the type of the component and should go last in the component's name:

HomeUsersView

Reply:
Plus you should NOT rename the component at import time. Let's keep 1:1 between component definition name and usage name